test/valid_field_test.exs

```diff
@@ -23,6 +23,11 @@ defmodule ValidFieldTest do
       ValidField.with_changeset(%Model{})
       |> ValidField.assert_invalid_field(:title, ["", nil, "Something else"])
     end
+
+    assert_raise ValidField.ValidationError, "Expected the following values to be invalid for \"date_of_birth\": #Ecto.Date<2016-08-23>", fn ->
```

Comment:
Ah! I think I'd have to change the generated error message and update the tests, docs, etc. since it's generating the error message with double quotes. 

Thoughts on keeping it as is for now to keep it consistent with the other tests and changing it all at once in another PR?